src/jsdoc/react/react-parser.ts

```diff
@@ -96,9 +98,11 @@ function stringifyType(prop: { name: string; value?: any; raw?: string }): strin
 export default async function parse(data: string, filePath?: PathOsBased): Promise<Doclet[] | undefined> {
   const doclets: Array<Doclet> = [];
   try {
-    const componentsInfo = reactDocs.parse(data, reactDocs.resolver.findAllComponentDefinitions, undefined, {
-      configFile: false
+    const componentsInfo = reactDocs.parse(data, reactDocs.resolver.findAllExportedComponentDefinitions, undefined, {
+      configFile: false,
+      filename: filePath // should we use pathNormalizeToLinux(filePath) ?
```

Comment:
@KutnerUri , no need for the `pathNormalizeToLinux` here. It's data we wend to react parse function, I believe they take care of it already.

Reply:
ok, I will remove the comment